vnext/src/Libraries/Components/Touchable/TouchableHighlight.windows.js

```diff
@@ -324,19 +324,18 @@ class TouchableHighlight extends React.Component<Props, State> {
         nextFocusRight={this.props.nextFocusRight}
         nextFocusUp={this.props.nextFocusUp}
         focusable={
-          this.props.focusable !== false && this.props.onPress !== undefined
+          (this.props.focusable !== false &&
+            this.props.onPress !== undefined) ||
+          ((this.props.acceptsKeyboardFocus === undefined ||
```

Comment:
Why do we need to check acceptsKeyboardFocus for this one? If we modify this could we wrap it with "// [Windows" like below?

Reply:
Same for other touchables

Reply:
I was trying to move away from acceptsKeyboardFocus, but since the property is still around and people can still use it, I'm combing the value into focusable to get the right behavior. 

We don't have to do this, I'll revert it and leave the file untouched to make it less confusing.

Reply:
We do want to set focusable by default, instead of acceptsKeyboardFocus. The part that I was curious about was setting focusable on the inner view dependent on acceptsKeyboardFocus instead of just focusable.

Reply:
I was trying to combine acceptsKeyboardFocus value into focusable, if you set either of them to true, focusable will be true, something like focusable = focusable || acceptsKeyboardFocus, but I think that's just more confusing so I reverted it.